CRAMv3.tex

```diff
@@ -691,43 +691,38 @@ \subsubsection*{Data series encodings}
 \subsubsection*{Tag encodings}
 \label{subsubsec:tags}
 
-The TL (tag list) data series represents combined information about the number 
-of tags in a record and their ids. 
-
-Let $L_{i}=\{T_{i0}, T_{i1}, \ldots, T_{ix-1}\}$
-be sorted list of all $x$ tag ids for a record $R_{i}$, where $i$ is the sequential
-record index and $T_{ij}$ denotes $j$-th tag id in the record.
-The list of unique $L_{i}$ is assigned sequential
-integer numbers starting with 0. These integer numbers represent the TL data series. 
-The list of unique $L_{i}$ is stored as the TD value in the preservation map, see section \ref{subsec:compression-header}.
+The tag dictionary describes the combinations of tag id / type that occur on each alignment record.
+For example if we search the id / types present in each record and find only two combinations -- X1:i BC:Z SA:Z: and X1:i: BC:Z -- then we have two dictionary entries in the TD map.
+
+Let $L_{i}=\{T_{i0}, T_{i1}, \ldots, T_{ix}\}$ be a list of all tag ids for a record $R_{i}$, where $i$ is the sequential record index and $T_{ij}$ denotes $j$-th tag id in the record.
+The list of unique $L_{i}$ is stored as the TD value in the preservation map.
```

Comment:
Should this be explicit about whether tag order counts (are different permutations of the same set of tags permitted to appear in the dictionary) ?

Reply:
Thanks for the background - I like the additional text you proposed.